test/smoke/polymer-loaded-twice.html

```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+  <link rel="import" href="../../polymer.html" />
+  <link rel="import" href="/polymer.html" />
```

Comment:
Use query parameters instead absolute url to make sure you can serve this source from a different endpoint as well.

Reply:
Load boot twice, verify error